__tests__/App.test.js

```diff
@@ -0,0 +1,3 @@
+it('renders correctly', () => {
```

Comment:
We always add a describe in root element 

```suggestion
describe(() => {
  it('renders correctly', () => {
  });
}):
```